includes/reader-activation/class-reader-activation.php

```diff
@@ -1226,6 +1226,13 @@ public static function render_subscription_lists_inputs( $lists = [], $checked =
 			$lists = \Newspack_Newsletters_Subscription::get_lists_config();
 		}
 
+		/**
+		 * Filter the available newsletter subscription lists in the Registration form.
+		 *
+		 * @param array[] $lists List config keyed by their ID.
+		 */
+		$lists = apply_filters( 'newspack_auth_form_available_newsletters_lists', $lists );
```

Comment:
```suggestion
		$lists = apply_filters( 'newspack_auth_form_newsletters_lists', $lists );
```

NIT: `available` seems redundant for this generic filter